addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1003,8 +1003,6 @@ def _get_allowed_message_partner_ids(self, partner_ids):
         return partners.ids
 
     def message_post(self, *, message_type="notification", partner_ids=None, **kwargs):
-        # sudo: discuss.channel - write to discuss.channel is not accessible for most users
-        self.sudo().last_interest_dt = fields.Datetime.now()
```

Comment:
Removing it from here will break properly bumping with all other "message post" flows.

The proper fix is likely to add a condition here to ignore notification type.